compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -482,23 +482,56 @@ private BLangConstantValue calculateMod(BLangConstantValue lhs, BLangConstantVal
         return new BLangConstantValue(result, currentConstSymbol.type);
     }
 
+    private Object calculateNegationForInt(BLangConstantValue value) {
```

Comment:
Is negation the correct term?

Reply:
According to the spec [[2]](https://ballerina.io/spec/lang/master/#section_6.24) it is the correct term right?